### DISCUSSION TITLE
Help for configuration

### DISCUSSION THREAD

**Discussion by Yota73** [edumeet/edumeet-client#175] [Q&A]
Hello and compliments for your amazing job!
I'm using the version 3 for a while without problems and now I would like to move to latest version.
I try to use a full docker-compose version and I can use it from PC to PC but if I connect with a mobile device the video is black and there is a waiting icon endless.
I also try with your test website usually.in with the PC and mobile and I have the same result (on the mobile I see the PC camera).

I need to customize the edumeet-client so I use the code from the repository.
For my customization I try, before all, to build the docker image from the code and replace the one of the docker-compose to rebuild the full stack but when I try to connect with the same configuration that works with docker even the PC-PC connection doesn-t work.
I get always the black video with the spinner.
In my LAN the PC-PC connection works but not the mobile.

Thanks a lot to everyone who can help me

**Comment by N7Remus** [comment]
Hi,

Do you have problem with a specific device? (android, ios ...)
Are you using wifi or cellural data?
Do you have a valid cert?
Do you see any usefull logs? (in edumeet-docker .env file there are debug enviroment variables that you can set to see full log)

**Reply by Yota73** [reply]
I use it only on 5 android devices (I don't have ios devices at the moment) and I try it with wifi and cellular data with the same result.
I use the full strict cloudflare certs.
For the log I mount the docker-compose not in daemon mode so I can see all logs but without errors.

I have an update for the source code version: If I connect three devices I can see the first and the third. But when the second device exit from the call, the connection between the first and the third drop (black screen with spinner)

**Comment by havfo** [comment]
@Yota73 usually.in was hard-coded to use h264 video codec. That has now been removed, can you please try again?

**Reply by Yota73** [reply]
Thank you for your explanation. 
I delete all and I download all the repo again and ... it works with the same config.
Now I'm trying to setup a new development environment to edit the client without breaking enything ... finger crossed.
Sorry for wasting your time.
